### PR TITLE
Increase max message size

### PR DESCRIPTION
**Pull request by felix-re**
Increased read buffer size to 262144 to match the message sizes from autobahn-testsuite. 
The read functions of buffered_socket return 0 now if a problem occures during go_reading, because the error-function handle that. Due to that the unittests has been adaptet and an uninitialized pointer is set now.

### CODE REVIEW

File: src/tests/buffered_socket_test.cpp [hunk 1]
```diff
@@ -703,9 +703,10 @@ BOOST_AUTO_TEST_CASE(test_read_exactly_called_twice)
 BOOST_AUTO_TEST_CASE(test_read_exactly_buffer_wrap)
 {
 	for (unsigned int chunk_size = 1; chunk_size <= CONFIG_MAX_MESSAGE_SIZE; chunk_size++) {
-		size_t chunks = (CONFIG_MAX_MESSAGE_SIZE / chunk_size) + 1; 
+		size_t chunks = (CONFIG_MAX_MESSAGE_SIZE / chunk_size) + 1;
 		char buffer[chunk_size * chunks];
 		::memset(buffer, 0, sizeof(buffer));
+		readbuffer=buffer;
```
Comment (gatzka) [inline src/tests/buffered_socket_test.cpp]: Please adhere to coding style. (readbuffer = buffer)

**Review by gatzka — CHANGES_REQUESTED** [review]
Overall good, please make the small requested change.

File: src/tests/buffered_socket_test.cpp (hunk 1 above)
Comment (felix-re) [reply src/tests/buffered_socket_test.cpp]: done

**Review by gatzka — APPROVED** [review]
Thanks!

**Pull request merged**